requirements.txt

```diff
@@ -7,10 +7,10 @@ lxml==4.9.2
 pygments==2.14.0
 python-dateutil==2.8.2
 pyyaml==6.0
-httpx[http2]==0.21.2
+httpx[http2]==0.23.3
 Brotli==1.0.9
 uvloop==0.17.0
-httpx-socks[asyncio]==0.7.2
+httpx-socks[asyncio]==0.7.5
 setproctitle==1.3.2
 redis==4.5.4
 markdown-it-py==2.2.0
```

Comment:
copy @dalf's comment from https://github.com/return42/searxng/commit/d86ec70877d24272c41f8c8e2ccde012791b9540#commitcomment-107513541 to here ..

> The important thing to test is the proxy configuration (http and socks). When I have tried with mitmproxy and `verify: ~/.mitmproxy/mitmproxy-ca-cert.cer` I had an error. It worked before.